pom.xml

```diff
@@ -96,6 +99,27 @@
             <dependency>
                 <groupId>org.glassfish.jersey.containers</groupId>
                 <artifactId>jersey-container-grizzly2-http</artifactId>
+                <version>${version.grizzly}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.glassfish.jersey.media</groupId>
+                <artifactId>jersey-media-json-jackson</artifactId>
+                <version>${version.grizzly}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.fasterxml.jackson.core</groupId>
+                <artifactId>jackson-core</artifactId>
+                <version>${version.jackson}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.fasterxml.jackson.dataformat</groupId>
+                <artifactId>jackson-dataformat-yaml</artifactId>
+                <version>${version.jackson}</version>
+            </dependency>
+
+            <dependency>
```

Comment:
Duplicate declaration of `org.glassfish.jersey.containers:jersey-container-grizzly2-http:jar` and `org.glassfish.jersey.media:jersey-media-json-jackson:jar`

Full maven warning:
```
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for io.mesosphere.mesos.frameworks:cassandra-framework:jar:0.1.0-SNAPSHOT
[WARNING] 'dependencyManagement.dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.glassfish.jersey.containers:jersey-container-grizzly2-http:jar -> version ${version.grizzly} vs 2.16 @ io.mesosphere.mesos.frameworks:cassandra:0.1.0-SNAPSHOT, /home/ben.whitehead/mesosphere/cassandra-mesos/pom.xml, line 120, column 25
[WARNING] 'dependencyManagement.dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.glassfish.jersey.media:jersey-media-json-jackson:jar -> version ${version.grizzly} vs 2.16 @ io.mesosphere.mesos.frameworks:cassandra:0.1.0-SNAPSHOT, /home/ben.whitehead/mesosphere/cassandra-mesos/pom.xml, line 125, column 25
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for io.mesosphere.mesos.frameworks:cassandra-executor:jar:0.1.0-SNAPSHOT
[WARNING] 'dependencyManagement.dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.glassfish.jersey.containers:jersey-container-grizzly2-http:jar -> version ${version.grizzly} vs 2.16 @ io.mesosphere.mesos.frameworks:cassandra:0.1.0-SNAPSHOT, /home/ben.whitehead/mesosphere/cassandra-mesos/pom.xml, line 120, column 25
[WARNING] 'dependencyManagement.dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.glassfish.jersey.media:jersey-media-json-jackson:jar -> version ${version.grizzly} vs 2.16 @ io.mesosphere.mesos.frameworks:cassandra:0.1.0-SNAPSHOT, /home/ben.whitehead/mesosphere/cassandra-mesos/pom.xml, line 125, column 25
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for io.mesosphere.mesos.frameworks:cassandra-scheduler:jar:0.1.0-SNAPSHOT
[WARNING] 'dependencyManagement.dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.glassfish.jersey.containers:jersey-container-grizzly2-http:jar -> version ${version.grizzly} vs 2.16 @ io.mesosphere.mesos.frameworks:cassandra:0.1.0-SNAPSHOT, /home/ben.whitehead/mesosphere/cassandra-mesos/pom.xml, line 120, column 25
[WARNING] 'dependencyManagement.dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.glassfish.jersey.media:jersey-media-json-jackson:jar -> version ${version.grizzly} vs 2.16 @ io.mesosphere.mesos.frameworks:cassandra:0.1.0-SNAPSHOT, /home/ben.whitehead/mesosphere/cassandra-mesos/pom.xml, line 125, column 25
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for io.mesosphere.mesos.frameworks:cassandra-mesos-model:jar:0.1.0-SNAPSHOT
[WARNING] 'dependencyManagement.dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.glassfish.jersey.containers:jersey-container-grizzly2-http:jar -> version ${version.grizzly} vs 2.16 @ io.mesosphere.mesos.frameworks:cassandra:0.1.0-SNAPSHOT, /home/ben.whitehead/mesosphere/cassandra-mesos/pom.xml, line 120, column 25
[WARNING] 'dependencyManagement.dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.glassfish.jersey.media:jersey-media-json-jackson:jar -> version ${version.grizzly} vs 2.16 @ io.mesosphere.mesos.frameworks:cassandra:0.1.0-SNAPSHOT, /home/ben.whitehead/mesosphere/cassandra-mesos/pom.xml, line 125, column 25
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for io.mesosphere.mesos.frameworks:cassandra:pom:0.1.0-SNAPSHOT
[WARNING] 'dependencyManagement.dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.glassfish.jersey.containers:jersey-container-grizzly2-http:jar -> version ${version.grizzly} vs 2.16 @ line 120, column 25
[WARNING] 'dependencyManagement.dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.glassfish.jersey.media:jersey-media-json-jackson:jar -> version ${version.grizzly} vs 2.16 @ line 125, column 25
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING]
```